### PR TITLE
🌱 add verify-release.sh

### PR DESCRIPTION
**Pull request by tuminoid**
Add `./hack/verify-release.sh` to check if the release process has been completed, and/or regular chores have been done before release. This script has been adapted from the one I wrote for CAPM3.

Another PR(s) will be created to fix the issues identified.

### CODE REVIEW

**Comment by tuminoid** [comment]
/override test-ubuntu-integration-main
/override test-centos-e2e-integration-main
Save the trees.

**Review by lentzi90 — COMMENTED** [review]
Seems to be working
/lgtm

File: hack/verify-release.sh [hunk 1]
```diff
@@ -0,0 +1,576 @@
...
+}
+
+check_input()
+{
+    echo "Checking input ..."
+
+    # check version is input without leading v, since we have extra annotated
+    # tags in history and it needs manually to be edited out
+    if [[ "${VERSION}" =~ ^v\d+ ]]; then
+        echo "FATAL: given version includes a leading v. Example: 1.5.0"
```
Comment (kashifest) [inline hack/verify-release.sh:215]: I saw a previous example uses `Example: 0.4.0` shall we use the same here to relate to BMO release ?

**Review by kashifest — COMMENTED** [review]
/approve
/hold
One nit inline.

File: hack/verify-release.sh (hunk 1 above)
Comment (tuminoid) [reply hack/verify-release.sh]: Good catch, fixed.

**Comment by tuminoid** [comment]
/override test-ubuntu-integration-main
/override test-centos-e2e-integration-main
Save the trees.

@lentzi90 please re-lgtm

**Comment by tuminoid** [comment]
/unhold
Nit was fixed.

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Pull request merged**